stac_fastapi/pgstac/stac_fastapi/pgstac/transactions.py

```diff
@@ -21,15 +22,20 @@
 @attr.s
 class TransactionsClient(AsyncBaseTransactionsClient):
     """Transactions extension specific CRUD operations."""
-
+    
     async def create_item(
         self, item: stac_types.Item, **kwargs
     ) -> Optional[Union[stac_types.Item, Response]]:
         """Create item."""
         request = kwargs["request"]
         pool = request.app.state.writepool
         await dbfunc(pool, "create_item", item)
-        return item
+        item["links"] = await ItemLinks(
+            collection_id=item["collection"],
```

Comment:
Just a note that if we make the `collection_id` path parameter available here, as suggested in #406 and #384 then we should update this to get the Collection ID from that parameter.

Reply:
> Just a note that if we make the `collection_id` path parameter available here, as suggested in #406 and #384 then we should update this to get the Collection ID from that parameter.

#425 makes this change, so we may want to merge that PR first if it is approved.